pandas/core/generic.py

```diff
@@ -2329,6 +2330,11 @@ def to_json(
 
             .. versionadded:: 0.23.0
 
+        indent : integer, default 0
```

Comment:
The parameter name doesn't seem 100% intuitive given the description.  Perhaps `indentLength` ?

Reply:
Indifferent though the stdlib uses indent as well, so naming consistency with that might have some advantages

Reply:
in io.formats.html, `indent_delta` is used, although this isn't yet part of the public API.

we should probably be consistent. so if `indent` is used here, would also need to update io.formats.html

Reply:
> Indifferent though the stdlib uses indent as well

They also use `ensure_ascii`, while we use `force_ascii` 😉

Reply:
i agree.